architecture/core_concepts/routes.adoc

```diff
@@ -1062,3 +1063,69 @@ The routers do not clear the `route status` field. To remove the stale entries
 in the route status, use the
 link:https://github.com/openshift/origin/blob/master/images/router/clear-route-status.sh[clear-route-status
 script].
+
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace ns1 creates the oldest route r1  www.abc.xyz, then it
+owns that host name and subdomain (abc.xyz) for wildcard routes.  If another
+namespace ns2 tries to create a route with say a different path www.abc.xyz/path1/path2,
+it would fail because a route in another namespace (ns1 in this case) "owns"
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.html#using-wildcard-routes[wildcard routes]
+as we now need to look at routes that are granted claims to hosts (*.abc.xyz)
+that belong to the subdomain (abc.xyz) associated with the wildcard route.
+
+The disable namespace ownership check relaxes those restrictions and allows
+hosts (and subdomains) to be claimed across namespaces.
+
+[WARNING]
+====
+If you decide to disable the namespace ownership checks in your router,
+please be aware that this allows end users to claim ownership of hosts
+across namespaces. Whilst this policy can be lax/relaxed in certain
+development environments, use this feature with caution in production
+environments and ensure that your cluster policy has locked down end
+users from creating routes.
+====
+
+For example, with ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true, if
+namespace ns1 creates the oldest route r1  www.abc.xyz,  it owns only
+the hostname (+ path).  Another namespace can create a wildcard route
+even though it doesn't have the oldest route in that subdomain (abc.xyz)
+and we could potentially have other namespaces claiming other
+non-wildcard overlapping hosts (for example, foo.abc.xyz, bar.abc.xyz,
+baz.abc.xyz) and their claims would be granted.
+
+Any other namespace (e.g. ns2) can now create
+a route r2  www.abc.xyz/p1/p2  and it would be admitted.  Similarly
+another namespace (ns3) can also create a route  wildthing.abc.xyz
+with a subdomain wildcard policy and it can own the wildcard.
+
+So the policy with ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK=true is more
+lax and allows claims across namespaces.  The only time the router would
+reject a route with the namespace ownership disabled is if the host+path
+is already claimed.
+
+For example: if a new route rx tries to claim www.abc.xyz/p1/p2  - that
```

Comment:
For example, if

    *_www.abc.xyz/p1/p2_* , then that would be rejected because